services/frontend-service/src/ui/App/index.tsx

```diff
@@ -23,6 +23,26 @@ import { PanicOverview, showSnackbarWarn, UpdateOverview, useReleaseDialog, useR
 import { useApi } from '../utils/GrpcApi';
 import { AzureAuthProvider, UpdateFrontendConfig, useAzureAuthSub } from '../utils/AzureAuthProvider';
 import { Snackbar } from '../components/snackbar/snackbar';
+import { mergeMap, retryWhen } from 'rxjs/operators';
+import { Observable, throwError, timer } from 'rxjs';
+
+// retry strategy: retries the observable subscription with randomized exponential backoff
+// source: https://www.learnrxjs.io/learn-rxjs/operators/error_handling/retrywhen#examples
+function retryStrategy(maxRetryAttempts = 3) {
+    return (attempts: Observable<any>): Observable<any> =>
+        attempts.pipe(
+            mergeMap((error, i) => {
+                // retry attempt number
+                const retryAttempt = i;
```

Comment:
You are never incrementing this here. Is that correct?

Reply:
this doesn't need incrementing. (I had to figure out counting from 0 / 1) I just decided eventually to just remove the i